src/xml.tsx

```diff
@@ -171,10 +171,15 @@ export class SvgFromXml extends Component<XmlProps, XmlState> {
   }
 
   parse(xml: string | null) {
+    const { onError = err } = this.props;
     try {
       this.setState({ ast: xml ? parse(xml) : null });
     } catch (e) {
```

Comment:
Maybe we could do `e: Error` here to not cast below?

Reply:
As far as I know, we can't catch error types, or is there something I'm missing?

Reply:
Oh, didn't know that. Then it's fine if we are sure that the `e` is of type `Error`.